main.go

```diff
@@ -8,14 +8,16 @@ import (
 	"os"
 	"strconv"
 	"time"
+	"strings"
 )
 
-const VERSION = "0.2.0"
+const VERSION = "0.3.0"
 
 var (
```

Comment:
app  = kingpin.New("your_program_name", "A command line tool for.....")
app.Flag()